bin/serialport-terminal.js

```diff
@@ -49,7 +49,7 @@ var openOptions = {
   stopBits: args.stopbits
 };
 
-var port = new serialport.SerialPort(args.port, openOptions);
+var port = new Serialport(args.port, openOptions);
```

Comment:
Just to mention, you might want to hunt for `Serialport` -> `SerialPort` for consistency